internal/env.go

```diff
@@ -41,3 +41,13 @@ func IntEnv(key string, def int) int {
 	}
 	return v
 }
+
+// StringEnv returns the value of an environment variable, or
+// def otherwise.
+func StringEnv(key string, def string) string {
```

Comment:
It is needed for this, otherwise the `defaultConfig` construction gets more ugly:

https://github.com/DataDog/dd-trace-go/pull/1511/files#diff-8f9f977d509d59fb878b9f086454a6b34dda3d6577d8a4568230af5b9dd8deb3R212

os.Getenv returns both a string and an err.

Reply:
`os.Getenv` just returns a string ([reference](https://pkg.go.dev/os#Getenv)), are you thinking of `os.LookupEnv`?

Reply:
ah yes, will fix